src/layouts/SiteDashboard/SiteDashboard.tsx

```diff
@@ -0,0 +1,244 @@
+import {
+  Box,
+  Container,
+  Flex,
+  Heading,
+  HStack,
+  Icon,
+  Link,
+  Skeleton,
+  Spacer,
+  Text,
+  VStack,
+} from "@chakra-ui/react"
+import { Button } from "@opengovsg/design-system-react"
+import { ButtonLink } from "components/ButtonLink"
+import {
+  DisplayCard,
+  DisplayCardCaption,
+  DisplayCardContent,
+  DisplayCardHeader,
+  DisplayCardTitle,
+} from "components/DisplayCard"
+import {
+  MenuDropdownButton,
+  MenuDropdownItem,
+} from "components/MenuDropdownButton"
+import { BiCheckCircle, BiCog, BiEditAlt, BiGroup } from "react-icons/bi"
+import { useParams, Link as RouterLink } from "react-router-dom"
+
+import {
+  useGetSiteInfo,
+  useGetReviewRequests,
+  useGetCollaboratorsStatistics,
+} from "hooks/siteDashboardHooks"
+
+import { SiteDashboardHumanImage } from "assets"
+
+import { SiteViewLayout } from "../layouts"
+
+import { CollaboratorsStatistics } from "./components/CollaboratorsStatistics"
+import { EmptyReviewRequest } from "./components/EmptyReviewRequest"
+import { ReviewRequestCard } from "./components/ReviewRequestCard"
+
+export const SiteDashboard = (): JSX.Element => {
+  const { siteName } = useParams<{ siteName: string }>()
+  const {
+    data: siteInfo,
+    isError: isSiteInfoError,
+    isLoading: isSiteInfoLoading,
+  } = useGetSiteInfo(siteName)
+  const {
+    data: reviewRequests,
+    isError: isReviewRequestsError,
+    isLoading: isReviewRequestsLoading,
+  } = useGetReviewRequests(siteName)
+  const {
+    data: collaboratorsStatistics,
+    isError: isCollaboratorsStatisticsError,
+    isLoading: isCollaboratorsStatisticsLoading,
+  } = useGetCollaboratorsStatistics(siteName)
+
+  return (
+    <SiteViewLayout overflow="hidden" px="10">
+      <Container maxW="container.xl" minH="100vh">
```

Comment:
It's to get the light blue background to cover the entire screen, otherwise the light blue background will only be shown for the portion where there is content, and will have a weird gap at the bottom of a different colour if the screen is large.